app/src/main/java/com/portfolio/tasky/entry/fragments/LoginFragment.kt

```diff
@@ -43,17 +47,17 @@ class LoginFragment : Fragment(), FragmentInflater by FragmentInflaterImpl(), Te
     }
 
     private fun setObservers() {
-        viewModel.emailChange.observe(viewLifecycleOwner){
+        viewModel.emailChange.observe(viewLifecycleOwner) {
```

Comment:
Instead of naming it `emailChange` I'd prefer something like just `email` since a LiveData doesn't reflect an email change really, but rather just holds the value of that email for which you can then subscribe to changes here.